lib/omnibus/software.rb

```diff
@@ -906,7 +906,12 @@ def source_type
     #
     def build_me
       if Config.use_git_caching
-        if project.dirty?
+        if !resolved_version
+          log.info(log_key) do
+            "Forcing a build because resolved version is nil"
+          end
+          execute_build
```

Comment:
I'll need to take a new look at that code.  I believe that execute_build will certainly build without the git cache and make a new project in the cache, but I don't see anywhere that would call project.dirty! to ensure that the following softwares also build without the cache.